src/app/services/task.service.ts

```diff
@@ -2,23 +2,23 @@ import {Injectable} from '@angular/core';
 import {HttpClient} from "@angular/common/http";
 import {TaskModel} from "../shared/task.model";
 import {BehaviorSubject, first, Observable} from "rxjs";
+import {localEnvironment} from "../environments/local";
```

Comment:
what happens if the environment is changed to test? this code block wont execute anymore and throw an error, you need to update angular.json configuration if you want to use the environment variables according to environment